packages/bouncer/extra/dcos_add_user.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+"""
+This script adds remote users in DC/OS IAM for the Auth0 identity provider.
+"""
+
+import argparse
+import logging
+import re
+
+import requests
+
+
+log = logging.getLogger(__name__)
+logging.basicConfig(format='[%(levelname)s] %(message)s', level='INFO')
+
+
+# To keep this script simple and avoid authentication and authorization this
+# script uses local IAM address instead of going through Admin Router
+IAM_BASE_URL = 'http://127.0.0.1:8101'
+
+
+def add_user(uid: str) -> None:
+    """
+    Create a user in DC/OS IAM:
+https://github.com/dcos/dcos/blob/abaeb5cceedd5661b8d96ff47f8bb5ef212afbdc/packages/dcos-integration-test/extra/test_legacy_user_management.py#L96
+    """
+    url = '{iam}/acs/api/v1/users/{uid}'.format(
+        iam=IAM_BASE_URL,
+        uid=uid,
+    )
+    r = requests.put(url, json={
+        'provider_type_str': 'oidc',
+        'provider_id': 'https://dcos.auth0.com/'
+        }
+    )
+
+    # The 409 response code means that user already exists in the DC/OS IAM
+    # service
+    if r.status_code == 409:
+        log.info('Skipping existing IAM user `%s`', uid)
```

Comment:
If we change this one more time, let's reword into something in the lines of "User with the given uid already exists".

The current message was copied from a different context where multiple users were processed. "Skipping" does not really make sense here.